elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -387,9 +387,13 @@ def extras_bugs(bugs: type_bug_set) -> type_bug_set:
 
 
 def print_report(bugs: type_bug_list, output: str = 'text') -> None:
+    approved_url = 'https://source.redhat.com/groups/public/openshift/openshift_wiki/openshift_bugzilla_process'
     if output == 'slack':
         for bug in bugs:
-            click.echo("<{}|{}> - {:<25s} ".format(bug.weburl, bug.id, bug.component))
+            if bug.release_blocker:
+                click.echo("<{}|_Release blocker: Approved_> bug for <{}|{}> - {:<25s} ".format(approved_url, bug.weburl, bug.id, bug.component))
```

Comment:
I don't know if this is the best place to implement this formatting, I would rather keep this in check_bugs.py

Reply:
Hmm but I see that we are relying on already this report to come pre-formatted .. what we can do is use `--output json` instead of `--output slack` for the elliott commands that we call in check_bugs.py and then proceed to do formatting as we want there.

Reply:
For better or worse, we are currently creating the markup in elliott, and not in pyartcd. Don't think there is a reason to change.